x/mint/client/rest/rest.go

```diff
@@ -1,14 +1,71 @@
 package rest
 
 import (
+	"net/http"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govrest "github.com/cosmos/cosmos-sdk/x/gov/client/rest"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 	"github.com/gorilla/mux"
+	"github.com/sei-protocol/sei-chain/x/mint/types"
 
 	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/client/rest"
+	clientrest "github.com/cosmos/cosmos-sdk/client/rest"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	typesrest "github.com/cosmos/cosmos-sdk/types/rest"
 )
 
 // RegisterRoutes registers minting module REST handlers on the provided router.
 func RegisterRoutes(clientCtx client.Context, rtr *mux.Router) {
-	r := rest.WithHTTPDeprecationHeaders(rtr)
+	r := clientrest.WithHTTPDeprecationHeaders(rtr)
 	registerQueryRoutes(clientCtx, r)
 }
+
+// PlanRequest defines a proposal for a new upgrade plan.
+type UpdateMinterRequest struct {
```

Comment:
Is this documented any where else for how to make a proposal?

Reply:
I'll add it as apart of the x/mint module readme